pets.js

```diff
@@ -11,6 +11,11 @@
 const pets = ['Cat', 'Dog', 'Hamster', 'Guinea Pig', 'Salmon']
 
 function main() {
+  const toPrint = pets.map((pet, index) => {
+    return `${index}. ${pet}`
```

Comment:
Index is starting from 0, so if you want to start with 1, add +1 to index

Reply:
thank you for your feedback